eiffel-vocabulary/EiffelServiceDeployedEvent.md

```diff
@@ -45,30 +45,162 @@ __Type:__ String
 __Required:__ No  
 __Description:__ A URI where further information can be obtained, if applicable. 
 
-## Legal Link Types
-### ENVIRONMENT
-__Legal targets:__ [EiffelEnvironmentDefinedEvent](https://github.com/Ericsson/eiffel/blob/master/eiffel-vocabulary/EiffelEnvironmentDefinedEvent.md)  
-__Multiple allowed:__ No  
-__Description:__ Identifies the environment in which an event occurred, e.g. in which environment an artifact was built or an issue was verified.
+## Legal Link Types 
+
+### ARTIFACT_DEPLOYED
+__Required:__ No  
```

Comment:
May be we could explain more on why required is no, like if the service allocator could find the matching service with all infrastructure already deployed we need not deploy anything

Reply:
Setting it to Yes, would simply add a strict constraint that usually is avoided as I see it.  But we can for sure discuss this